feature-libs/product-configurator/rulebased/components/group-title/configurator-group-title.component.html

```diff
@@ -1,5 +1,10 @@
 <ng-container *ngIf="displayedGroup$ | async as group; else ghostGroup">
   <div class="cx-group-title">
+    <ng-container *ngIf="isMobile() | async as mobile">
```

Comment:
Why do we still have the preHeader slot in feature-libs/product-configurator/rulebased/root/variant/variant-configurator-interactive-layout.module.ts? I guess it can be removed.
(And I don't understand why are we not seeing the hamburger menu twice? It is visible in the DOM)

Reply:
It's true. We till need PreHeader to be able to close the menu because closing button **X** stays in PreHeader. (see attached screenshot) 
![image](https://user-images.githubusercontent.com/61147963/208030922-1cc74254-3c6e-4681-ae2a-717182a13657.png)

Reply:
That's why the visibility of PreHeader will be accordingly adjusted depending on whether the hamburger menu is expanded or not.